cmd/containerboot/main_test.go

```diff
@@ -1217,6 +1234,12 @@ func (l *localAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 		w.Write([]byte("fake metrics"))
 		return
+	case "/localapi/v0/status":
```

Comment:
Is this needed? I can't remember a need for it from the code in this PR but may be forgetting something.

Reply:
Lleftover from when this PR was still doing the tailnet target pings based on netmap status